core/Config.php

```diff
@@ -514,22 +514,23 @@ protected function getUncached($class, $name, $sourceOptions, &$result, $suppres
 			if ($extraSources) $sources = array_merge($sources, $extraSources);
 		}
 
-		$value = $nothing = null;
-
 		foreach ($sources as $staticSource) {
-			if (is_array($staticSource)) {
-				$value = isset($staticSource[$name]) ? $staticSource[$name] : $nothing;
-			}
-			else {
-				foreach ($this->staticManifests as $i => $statics) {
-					$value = $statics->get($staticSource, $name, $nothing);
-					if ($value !== $nothing) break;
+			if(is_array($staticSource)) {
+				$value = isset($staticSource[$name]) ? $staticSource[$name] : null;
+			} else if(class_exists($staticSource) && property_exists($staticSource, $name)) {
```

Comment:
I think the `class_exists()` check here is redundant: `property_exists()` will return false if the class doesn’t exist.